src/pages/LendingStats/LendingStatsPage.vue

```diff
@@ -3,58 +3,62 @@
 		<template #secondary>
 			<the-my-kiva-secondary-menu />
 		</template>
-		<div class="row page-content">
-			<the-portfolio-tertiary-menu class="show-for-large" />
-			<div class="small-12 large-9 columns">
-				<h1>Lending stats</h1>
-				<p>
-					This is a snapshot of your lending activity on Kiva.
-					Use this page to collect loans and hit milestones along the way.
-				</p>
-				<hr>
-				<stats-section
-					title="Countries &amp; Territories*"
-					noun="country"
-					:not-lent-to="countriesNotLentTo"
-					:lent-to="countriesLentTo"
-					:total="totalCountries"
-					item-key="isoCode"
-					unlent-url="/lend/countries-not-lent"
-				/>
-				<hr>
-				<stats-section
-					title="Sectors"
-					noun="sector"
-					:not-lent-to="sectorsNotLentTo"
-					:lent-to="sectorsLentTo"
-					:icon-key="iconForSector"
-				/>
-				<hr>
-				<stats-section
-					title="Activities"
-					noun="activity"
-					:not-lent-to="activitiesNotLentTo"
-					:lent-to="activitiesLentTo"
-				/>
-				<hr>
-				<stats-section
-					title="Field Partners*"
-					noun="Field Partner"
-					:not-lent-to="partnersNotLentTo"
-					:lent-to="partnersLentTo"
-					:total="totalPartners"
-					query="partner"
-				/>
-				<hr>
-				<p>
-					* Please note, Kiva is continually adding and ending partnerships as we deem necessary.
-					This means, you may end up supporting a loan in a country or through a Field Partner that
-					is no longer active and therefore not included in the total number of countries or partners
-					noted on this page. For this reason, it is possible to see discrepancies between the number
-					you've supported and the number to go.
-				</p>
+		<kv-default-wrapper>
```

Comment:
Wondering if this makes this page drastically different in layout than the other portfolio pages???

Reply:
Yeah, it's a little different for sure. Though this whole area is not particularly consistent. What do you think?

<img width="1190" alt="Screen Shot 2021-11-15 at 5 46 47 PM" src="https://user-images.githubusercontent.com/187937/141880985-7251c1b8-57f9-405d-a4cf-b05984f4adf6.png">

<img width="1222" alt="Screen Shot 2021-11-15 at 5 46 35 PM" src="https://user-images.githubusercontent.com/187937/141880978-3f89026c-c1e6-46aa-977b-6e0aa1070175.png">

<img width="1222" alt="Screen Shot 2021-11-15 at 5 47 18 PM" src="https://user-images.githubusercontent.com/187937/141880990-b2146fb8-550f-429d-9b22-4566c00f7222.png">

<img width="1174" alt="Screen Shot 2021-11-15 at 5 47 01 PM" src="https://user-images.githubusercontent.com/187937/141880988-5625589e-5f39-4964-b17b-69637a2ac6e1.png">

Reply:
Looks fine. I was mostly thinking of the width of the container switching up a new way. The extra space at the top seems a little much but we can go with it...